rascal-lsp/src/main/rascal/lang/rascal/lsp/refactor/rename/Fields.rsc

```diff
@@ -95,21 +95,6 @@ set[Define] getFieldDefinitions(Tree container, str fieldName, TModel tm, TModel
     return {};
 }
 
-set[Define] getFieldDefinitions(Tree container, str fieldName, TModel tm, TModel(loc) getModel)
```

Comment:
so, new features with only deleting a function? that is a rich PR that I gladly approve 👍🏼